tools/config-generator/main.go

```diff
@@ -130,6 +130,26 @@ type baseProwJobTemplateData struct {
 // ################ data definitions that are used for the prow config file generation ################
 // ####################################################################################################
 
+// outputter is a struct that directs program output and counts the number of write calls.
+type outputter struct {
```

Comment:
You can remove "sink" and have `outputter` itself be an io.Writer; like I did [here](https://github.com/knative/test-infra/blob/master/pkg/interactive/docker.go#L40)

Reply:
which would clean up the code a bit

Reply:
Done.